src/website/app/pages/GettingStarted/gettingStarted.md

```diff
@@ -1,14 +1,24 @@
 # Getting Started
 
-We made Mineral UI to help you quickly build React apps with a
-high-quality UI.
+Mineral UI helps you quickly build React apps with high-quality, accessible components.
+Use npm or yarn to get components and themes that have been tested across modern browsers.
+
+Below are a few simple steps to apply consistent styling and hierarchy to your app,
+so you can focus on solving business problems.
+Mineral UI is an open source project and welcomes all contributions.
 
 ## Installation
 
 Install the [Mineral UI package](https://www.npmjs.com/package/mineral-ui):
 
 ```bash
-npm install --save mineral-ui
+$ npm install --save mineral-ui
```

Comment:
Long-term, I would like to find a way to include the `$` as a non-selectable element somehow, so that it's easy to copy/paste and run the command. Not at all a blocker here, of course.

Reply:
agreed. I wonder if we could do something like check the language and add an unselectable '$ automatically to `bash` types